multimedia/M2VDownsizer/Files/1003-Patch-config.h.diff

```diff
@@ -0,0 +1,251 @@
+--- config.h.orig	2006-02-27 21:38:49.000000000 -0500
++++ config.h	2022-08-06 07:34:47.000000000 -0400
```

Comment:
This patch is much larger than I would have expected to be necessary. All I would have expected to need is
```
#ifndef CONFIG_H
#define CONFIG_H
```
at the top and
```
#endif
```
at the bottom.

Reply:
Yes, it is overkill as I mentioned in one of my posts to the dev mailing list.  The first include guard is probably all that is needed.  But all the other include guards will not slow down the execution of the final binary given how the CPP works.

Reply:
Ryan, I cleaned up `config.h` and eliminated most of the include guards.  M2VDownsizer build just fine.

See https://pastebin.com/K5ian88P

And I finally found the bug in config.h.  HAVE_STDINT_H was defined twice in config.h.   Once I cleaned up the original formatting, it was easier to spot.  I do not know how I missed it the first time!

I recommend that we go with the updated config.h.    I could EMAIL the file to you or you could just use the pastbin.  I will likely be away from my development computer for a few days so pushing a new update to the PR may be difficult.

Reply:
Here is the pastebin for the updated file - 1003-Patch-config.h.diff

https://pastebin.com/SFtxvnUu

Reply:
You may use whatever filenames and capitalizations within the files directory that you wish, provided you reference them from the Portfile with the same names and capitalizations.

Reply:
This issue has been resolved in my last push.  The patch file - **1003-Patch-config.h.diff** - has been revised.  All the unnecessary include guards have been removed from config.h